system/otherLoadTest/playlist.xml

```diff
@@ -17,6 +17,20 @@
 			<group>system</group>
 		</groups>
 	</test>
+	<test>
+		<testCaseName>system_custom</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(SYSTEMTEST_CMD_TEMPLATE) $(CUSTOM_TARGET);\
```

Comment:
We need to set a default value for `CUSTOM_TARGET `. Otherwise, the build without `CUSTOM_TARGET ` (i.e., regular build) will fail.

Reply:
Ah.. that was missed. 

Added! 

Ran some more tests with the default value: 

When `CUSTOM_TARGET` field is empty : https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/6495/console 
When `CUSTOM_TARGET` has a value given by user: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/6496/console

They look all good.